### ISSUE TITLE
indices element out of data bounds

### ISSUE THREAD

**Issue by sandeepnmenon** [onnx/tensorflow-onnx#704]
**Bug Description**
We have a query annotation model written in tensorflow and deployed as the onnx model. In our telemetry logs, we are seeing the following error come up.
[ONNXRuntimeError] : 2 : INVALID_ARGUMENT : indices element out of data bounds, idx=6157 data_dim=6148

The issue is happening only for a small percent of the queries and not for all the queries.

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows 10
- Tensorflow Version: 1.2
- Python version: 3.6

**Comment by jignparm** [comment]
Can you attach some information to reproduce the error (preferably using a small/medium-sized model)?

**Comment by guschmue** [comment]
ONNXRuntimeError ? Should this issue be filed to https://github.com/microsoft/onnxruntime ?

**Comment by jignparm** [comment]
The error message seems to be coming from OnnxRuntime, but the source may still be tf2onnx (e.g. some attribute of a node is set incorrectly).

Can you please provide details to reproduce?

**Comment by guschmue** [comment]
It is not impossible but a ton more likely that it is a onnxruntime issue and to get the right people looking at this I suggest opening the issue on  https://github.com/microsoft/onnxruntime

**Comment by sandeepnmenon** [comment]
Correction in bug
Pytorch was used to develop the model and export it to Onnx, rather than Tensorflow.
Please close the issue if it is not relevant to this repo.
I will raise an error on onnxruntime

**Comment by guschmue** [comment]
Yes, https://github.com/microsoft/onnxruntime is the right repo to file the issue.
